### DISCUSSION TITLE
Feature request for workflow logs: Always keep last workflow run log

### DISCUSSION THREAD

**Discussion by na-jakobs** [community/community#17672] [Actions]
When viewing old projects it's a hassle to not see any logs for the last workflow run if they have expired. Sometimes we need
to look at old stuff and figure out how it ran and what was done.

Can you add this feature/option please? 🙏

[ x ] Always keep last workflow logs despite expiry settings

**Comment by flying7eleven** [comment]
I think this would be a quite valuable feature for a lot of GitHub Action users. In my case I have services which are rolled out infrequently (several month between rollouts) through GitHub Actions. If something fails in a rollout it was helpful to view the log of the last successful rollout to fix the issue. I support this feature request!
